__tests__/boot.ts

```diff
@@ -16,6 +16,8 @@ import {
   Feature,
   FeatureType,
   MachineSource,
+  MarketingCta,
```

Comment:
Just something I want to streamline between apps and api 
I called it `MarketingCTA` as CTA is basically shortcut already.
But let me know which one we prefer 😅

Reply:
I started with `MarketingCTA`, then went to `MarketingCta`. Mostly because we write `ID` as `Id`.


Personally I like the use the right capitalisation, but it looks so wrong if you need to write things like `MarketingCTAId`, like where is the abbreviation starting or ending 😬  So it only works when there's spaces separation the words